### PR TITLE
DEVPROD-952: Spruce Hosts page docs

### PR DESCRIPTION
**Pull request by SupaJoon**
DEVPROD-952

### CODE REVIEW

File: docs/Hosts/Spawn-Hosts.md [hunk 1]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
 If you'd like to get a notification before a host expires, you can [set up a
 notification](../Project-Configuration/Notifications.md#spawn-host-expiration) for it.
+
+## Hosts Page
+
+The spruce hosts page shows all hosts and offers three batch actions applicable to hosts:
+
+1. Update Status
+   You can force a state change to these statuses:
+   -Decomissioned: Terminate a host running tasks after it's done running its current task.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:64]:
```suggestion
   - Decommissioned: Terminate a host after it's done running its current task.
```

File: docs/Hosts/Spawn-Hosts.md [hunk 2]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+   -Quarantined: Stop a host from running tasks without terminating it or shutting it down.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:65]:
```suggestion
   - Quarantined: Stop a host from running tasks without terminating it.
```
(q: what makes this different from the plain "Stopped"? It is worth noting here or down below if possible)

File: docs/Hosts/Spawn-Hosts.md [hunk 3]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+   -Terminate: Actually terminate the host.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:66]:
```suggestion
   - Terminate: Shut down the host.
```

File: docs/Hosts/Spawn-Hosts.md [hunk 4]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+   -Stopped: Stop it.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:67]:
```suggestion
   - Stopped: Stop the host.
```

File: docs/Hosts/Spawn-Hosts.md [hunk 5]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+   -Runnning: Start it back up.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:68]:
```suggestion
   - Running: Start the host.
```

File: docs/Hosts/Spawn-Hosts.md [hunk 6]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
 
 If you'd like to get a notification before a host expires, you can [set up a
 notification](../Project-Configuration/Notifications.md#spawn-host-expiration) for it.
+
+## Hosts Page
+
+The spruce hosts page shows all hosts and offers three batch actions applicable to hosts:
+
+1. Update Status
+   You can force a state change to these statuses:
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:63]:
```suggestion
1. Update Status

   You can force a state change to these statuses:
```
needs an extra line break for the intended formatting

File: docs/Hosts/Spawn-Hosts.md [hunk 7]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+   -Decomissioned: Terminate a host running tasks after it's done running its current task.
+   -Quarantined: Stop a host from running tasks without terminating it or shutting it down.
+   -Terminate: Actually terminate the host.
+   -Stopped: Stop it.
+   -Runnning: Start it back up.
+
+2. Restart Jasper
+   Try forcing the Evergreen agent (which runs in a system process called Jasper) to start back up in a way that's less disruptive than just rebooting the computer.
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:71]:
```suggestion
2. Restart Jasper

   This option will try forcing the Evergreen agent (which runs in a system process called Jasper) to start back up in a way that's less disruptive than just rebooting the host.
```
needs an extra line break for the intended formatting

File: docs/Hosts/Spawn-Hosts.md [hunk 8]
```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
...
+
+3. Reprovision
+   Hosts need to have a few starter files on the file system before they can run tasks, and pressing that button gives them the files they need if they're a static host(because there's no way to auto-populate them with the files they need). This option is useful for static hosts that get extremely borked (like the file system is corrupted and it loses all the data).
```
Comment (minnakt) [inline docs/Hosts/Spawn-Hosts.md:74]:
```suggestion
3. Reprovision

   Hosts need to have a few starter files on the file system before they can run tasks. Sometimes static hosts can get into bad states (e.g. the file system is corrupted) and stop functioning correctly. Reprovisioning a host will repopulate these files for static hosts.
```
needs an extra line break for the intended formatting & changed some wording here

**Review by ablack12 — APPROVED** [review]
LGTM with minna's suggestions 
(It seems like several comments are around ensuring the intended formatting applies -- you can see how your changes will look by clicking "View File")

image

**Pull request merged**